README.md

```diff
@@ -43,9 +42,6 @@ PostgresSQL
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] [OPTIONAL]
Kindly ensure to streamline the highlighted portions of your ReadMe so as to make it stand out better!

![image](https://user-images.githubusercontent.com/12745474/230955789-e488fc9b-7e02-4bbd-8603-a1839289caad.png)